CODEREVIEW

```diff
@@ -0,0 +1,5 @@
+// @CODEREVIEW: General remarks:
+// @CODEREVIEW:
+// @CODEREVIEW: Remove all unused code (code in comments)
+// @CODEREVIEW: Please run `phpcs` and get rid of all coding standard errors and warnings.
```

Comment:
@stanbellcom Regarding "Only string literals should be passed to t() where possible": You could consider translating the error `description`s in `lib/nemid-error-codes.php` to Danish (maybe even copy it to `nemid-error-codes.da.php` and load it based on the current locale). You say that ”Dagsorden is single language application” and since the error `text`s are in Danish so should the `descriptions` be.

There are good reasons for not using variables inside `t` (see https://drupal.stackexchange.com/a/9366), but I agree with you that it's not a major issue in this case.